imports/blocks/add-schedule/Layout.js

```diff
@@ -1,147 +1,195 @@
-import { Component, PropTypes } from "react";
+// @flow
+
+// $FlowMeteor
+import { Meteor } from "meteor/meteor";
 import Moment from "moment";
 import { css } from "aphrodite";
+import { Component } from "react";
 
+import TertiaryPhrase from "./phrases";
 import Forms from "../../components/forms";
 import CheckoutButtons from "../checkout-buttons";
 import Styles from "./styles-css";
 
-export default class Layout extends Component {
-
-  static propTypes = {
-    schedules: PropTypes.array,
-    save: PropTypes.func,
-    accounts: PropTypes.array, // eslint-disable-line
-    setFund: PropTypes.func,
-    state: PropTypes.object,
-    format: PropTypes.func,
-    total: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    saveDate: PropTypes.func,
-    setFrequency: PropTypes.func,
-    existing: PropTypes.string,
-    ready: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]), // XXX fix
-    text: PropTypes.string,
-    onSubmitSchedule: PropTypes.func,
-    dataId: PropTypes.string,
-  }
+type ILayout = {
+    accounts: Object[],
+    existing: Object,
+    format: Function,
+    onSubmitSchedule: Function,
+    ready: boolean,
+    save: Function,
+    saveDate: Function,
+    schedules: Object[],
+    setFrequency: Function,
+    setFund: Function,
+    state: Object,
+    text: string,
+    total: number,
+};
+
+// this definition of Layout doesn't work
+// const Layout = ({
+//   accounts,
+//   existing,
+//   format,
+//   onSubmitSchedule,
+//   ready,
+//   save,
+//   saveDate,
+//   schedules,
+//   setFrequency,
+//   setFund,
+//   state,
+//   text,
+//   total,
+// }: ILayout) => {
+
+// this definition of Layout works
+class Layout extends Component {
+
+  propTypes: ILayout;
 
   render() {
-    let { total } = this.props;
     const {
+      accounts,
+      existing,
+      format,
+      onSubmitSchedule,
+      ready,
+      save,
+      saveDate,
       schedules,
       setFrequency,
-      accounts,
       setFund,
       state,
-      save,
-      format,
-      saveDate,
-      existing,
-      ready,
+      text,
+      total,
     } = this.props;
 
-    if (!total) {
-      total = 0;
+/* end flip flop comment section */
+    const formClasses = [
+      "flush-bottom",
+      "h3",
+      "hard-top",
+      "outlined--dotted",
+      "outlined--light",
+    ];
+
+    let disableCheckout = false;
+    if (!total || total <= 0 || !ready) {
+      disableCheckout = true;
+    }
+
+    if (!accounts || accounts.length === 0) {
+      return null;
     }
 
     let prefillFund = accounts[0].value;
     if (existing && existing.details && existing.details.length && existing.details[0].account) {
       prefillFund = existing.details[0].account.id;
     }
 
-    // let defaultDate = Moment().add(1, "days")
-    let defaultDate;
+  // XXX: this was already commented out, should it be?
```

Comment:
@dcs619 funny story. When we were setting the default date, people clicked through and set up schedules too fast and missed the part of setting their own start date. So we removed it to slow down  and require people to choose their date